content/en/network_performance_monitoring/installation.md

```diff
@@ -44,6 +44,8 @@ The following **provisioning systems** are supported:
 - Chef 12.7+: See the [Datadog Chef recipe][6]
 - Ansible 2.6+: See the [Datadog Ansible role][7]
 
+**Istio** versions v1.6.4 and above are supported by [Datadog Agent v7.24.1+][1].
```

Comment:
copy looks good! can we move this above the line on Cilium?

Reply:
Sure